.github/workflows/latexmk.yml

```diff
@@ -15,7 +15,8 @@ jobs:
             examples
           latexmk_use_xelatex: true
       - uses: fregante/setup-git-user@v1
-      - name: Upload PDF to examples branch
+      - if: github.event_name == 'push'
```

Comment:
this makes sure that the example branch is not updated by PRs (in progress), and only updated when changes are pushed to the repo.

Reply:
Good to know!

Please let me know if you would like this to be merged

Thanks again!